framework-plugins/lisk-framework-chain-connector-plugin/README.md

```diff
@@ -0,0 +1,40 @@
+# @liskhq/lisk-framework-chain-connector-plugin
+
+@liskhq/lisk-framework-chain-connector-plugin is a plugin for creating and sending Cross-Chain-Update (CCU) Transactions.
```

Comment:
Maybe we can add a bit more here? Perhaps we can mention interoperability and connecting sidechain -> mainchain or sidechain -> sidechain